plugins/transformer/basic/client/recordupdater/record_updater.go

```diff
@@ -186,3 +252,55 @@ func (*RecordUpdater) obfuscateColumn(column arrow.Array) arrow.Array {
 	}
 	return bld.NewStringArray()
 }
+
+func (*RecordUpdater) obfuscateJSONColumns(column arrow.Array, jcs []jsonColumn) arrow.Array {
+	bld := types.NewJSONBuilder(array.NewExtensionBuilder(memory.DefaultAllocator, types.NewJSONType()))
+	for i := 0; i < column.Len(); i++ {
+		if !column.IsValid(i) {
+			bld.AppendNull()
+			continue
+		}
+
+		str := column.ValueStr(i)
+		for _, jc := range jcs {
+			val := gjson.Get(column.ValueStr(i), jc.columnPath)
+			if val.Exists() && val.Type == gjson.String {
+				if modified, err := sjson.Set(str, jc.columnPath, fmt.Sprintf("%x", sha256.Sum256([]byte(val.Str)))); err == nil {
+					str = modified
+					continue
+				}
+			}
+			str = val.Str
```

Comment:
```suggestion
```

Not sure we want to override the original JSON when the value doesn't exist.

I'm getting an error:
```
pgerror: severity: ERROR, code: 22P02, message: invalid input syntax for type json, detail :The input string ended unexpectedly., hint: , position: 0, internal_position: 0
```

on the spec below:

```yaml
kind: source
spec:
  name: "azure"
  path: "cloudquery/azure"
  version: "v15.0.0"
  destinations: ["postgresql"]
  tables: ["azure_advisor_recommendations"]
---
kind: destination
spec:
  name: "postgresql"
  path: "cloudquery/postgresql"
  version: "v8.6.3"
  write_mode: "overwrite-delete-stale"
  transformers: ["basic"]
  spec:
    connection_string: "user=postgres password=pass host=localhost port=5442 dbname=postgres sslmode=disable"
---
kind: transformer
spec:
  name: "basic"
  registry: grpc
  path: localhost:9999
  spec:
    transformations:
      - kind: obfuscate_columns
        tables: ["azure_advisor_recommendations"]
        columns: [
            "properties.extendedProperties.score",
          ]
```